activation/handler_v1.go

```diff
@@ -428,57 +428,11 @@ func (h *HandlerV1) checkWrongPrevAtx(
 				"registering at PoET",
 			log.ZContext(ctx),
 			zap.Stringer("smesher", atx.SmesherID),
-			log.ZShortStringer("expected", prevID),
```

Comment:
I think this info is quite helpful in the case this error happened. Getting it is just a single `atxs.PrevIDByNodeID` call within the same transaction 🙂

Reply:
The problem is it would require another db query to get the latest ATX of the smesher to provide it :/

Reply:
See my other comments - querying for the expected previous ATX first (which should be the only query in 99% of all cases) also makes the code easier and the second query simpler.